### ISSUE TITLE
RuntimeError: invalid hash value running command 8 of 01_apply_style_transfer_locally

### ISSUE THREAD

**Issue by roalexan** [Azure/batch-scoring-for-dl-models#25]
**Summary**
After running through all the setup steps on my Ubuntu 16.04 VM, I hit an error running command 8 in 01_apply_style_transfer_locally.

**Output**
Downloading: "https://download.pytorch.org/models/vgg19-dcbb9e9d.pth" to /home/roalexan/.torch/models/vgg19-dcbb9e9d.pth
9.0%
Traceback (most recent call last):
  File "style_transfer_script.py", line 402, in <module>
    cnn = models.vgg19(pretrained=True).features.to(device).eval()
  File "/home/roalexan/miniconda3/envs/batchscoringdl/lib/python3.6/site-packages/torchvision/models/vgg.py", line 180, in vgg19
    model.load_state_dict(model_zoo.load_url(model_urls['vgg19']))
  File "/home/roalexan/miniconda3/envs/batchscoringdl/lib/python3.6/site-packages/torch/utils/model_zoo.py", line 65, in load_url
    _download_url_to_file(url, cached_file, hash_prefix, progress=progress)
  File "/home/roalexan/miniconda3/envs/batchscoringdl/lib/python3.6/site-packages/torch/utils/model_zoo.py", line 101, in _download_url_to_file
    .format(hash_prefix, digest))
RuntimeError: invalid hash value (expected "dcbb9e9d", got "702d1061488da4c1dc1477021bbfc01c6b38597f526a6b1bd46e8b739ef98078")

**Comment by jiata** [comment]
It looks like it failed to download the pretrained vgg19 model. Maybe a connection issue.

Can you try deleting the downloaded file (which should be in ~/.torch/models) and try again?

**Comment by roalexan** [comment]
I reran, and this time it worked. My suspicion is that this can be traced back to installing ffmpeg, as the first time I did this I saw the below - didn't see it in the most recent attempt. Guess I'll go ahead and close this issue.

sudo add-apt-repository ppa:jonathonf/ffmpeg-3
sudo add-apt-repository ppa:jonathonf/tesseract
sudo apt-get update

W: Failed to fetch https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64/InRelease  The following signatures were invalid: KEYEXPIRED 1538166745  KEYEXPIRED 1538166745  KEYEXPIRED 1538166745  KEYEXPIRED 1538166745
W: Failed to fetch https://nvidia.github.io/nvidia-docker/ubuntu16.04/amd64/InRelease  The following signatures were invalid: KEYEXPIRED 1538166745  KEYEXPIRED 1538166745  KEYEXPIRED 1538166745  KEYEXPIRED 1538166745`
